htdocs/core/class/commonobject.class.php

```diff
@@ -235,6 +235,11 @@ abstract class CommonObject
 	 */
 	public $ref;
 
+	/**
+	 * @var string The object's old reference before update
+	 */
+	public $oldref;
```

Comment:
May be we can set this as deprecated because now, we should try to store old value using ->oldcopy=old object clone ?